documentation/docs/25-build-and-deploy/40-adapter-node.md

```diff
@@ -162,6 +170,47 @@ MY_CUSTOM_ORIGIN=https://my.site \
 node build
 ```
 
+> [Socket activation](#socket-activation) does not work when a prefix is specified because `LISTEN_PID` and `LISTEN_FDS` are set by the OS and cannot be renamed.
+
+## Graceful shutdown
+
+By default `adapter-node` gracefully shuts down the HTTP server when a `SIGTERM` or `SIGINT` signal is received. It does this by calling [`server.close`](https://nodejs.org/api/http.html#serverclosecallback), [`server.closeIdleConnections`](https://nodejs.org/api/http.html#servercloseidleconnections) and [`server.closeAllConnections`](https://nodejs.org/api/http.html#servercloseallconnections) under the hood. This will
+
+1. wait for requests that have already been made but not received a response yet to finish
+2. reject new requests
+3. close connections once they become idle
+4. and finally, close any remaining connections that are still active after [`SHUTDOWN_TIMEOUT`](#shutdown_timeout) seconds.
```

Comment:
Without looking at the code, I'd kind of expect the order of 1. and 2. here to be reversed, so not sure if I'm wrong about that?

What do you think about including the links with the relevant bullet points:
```suggestion
By default `adapter-node` gracefully shuts down the HTTP server when a `SIGTERM` or `SIGINT` signal is received. It will:

1. reject new requests ([`server.close`](https://nodejs.org/api/http.html#serverclosecallback))
2. wait for requests that have already been made but not received a response yet to finish and close connections once they become idle ([`server.closeIdleConnections`](https://nodejs.org/api/http.html#servercloseidleconnections))
3. and finally, close any remaining connections that are still active after [`SHUTDOWN_TIMEOUT`](#shutdown_timeout) seconds. ([`server.closeAllConnections`](https://nodejs.org/api/http.html#servercloseallconnections))
```

Reply:
Yeah, this makes sense.